meta-ostro-xt/classes/ostro-xt-image.bbclass

```diff
@@ -65,3 +67,6 @@ IMAGE_INSTALL += "${IMAGE_INSTALL_THERMAL_DAEMON}"
 
 # add package of Joule hardware specific udev rules
 IMAGE_INSTALL += "joule-udev-rules"
+
+# alsa-utils should support udev
+PACKAGECONFIG_append_pn-alsa-utils = " udev"
```

Comment:
looks like udev is default too. It should be safe to drop both PACKAGECONFIG_append lines